<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumePlayer</class>
 <widget class="QDockWidget" name="VolumePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>355</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFeatureMask</set>
  </property>
  <property name="windowTitle">
   <string>Volume Player</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="VolumePlayerWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="VolumePlayerLayout" stretch="0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="PlayerUpperLayout">
           <item>
            <widget class="QSlider" name="volumeSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="PlayerLowerLayout">
           <item>
            <layout class="QHBoxLayout" name="ButtonsLayout">
             <item>
              <widget class="QCheckBox" name="blending">
               <property name="text">
                <string>Blending</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="playVolumeButton">
               <property name="text">
                <string>Play</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pauseVolumeButton">
               <property name="text">
                <string>Pause</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stopVolumeButton">
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="previousVolumeButton">
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nextVolumeButton">
               <property name="text">
                <string>Next</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="SpeedLayout">
             <item>
              <widget class="QSlider" name="speedSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Speed:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="speedValue">
                 <property name="text">
                  <string>[val]</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="setMaxSpeedButton">
             <property name="text">
              <string>set speed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="applyForAllButton">
             <property name="text">
              <string>Apply transferfunction
for all</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="dataTable">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
